components/VM_Arm/src/main.c

```diff
@@ -633,7 +656,17 @@ int install_vm_devices(vm_t *vm)
     /* Install virtual devices */
     if (config_set(CONFIG_VM_PCI_SUPPORT)) {
         err = vm_install_vpci(vm, io_ports, pci);
-        assert(!err);
+        if (err) {
+            ZF_LOGE("Failed to install VPCI device (%d)", err);
+            return -1;
+        }
+    }
+
+    /* Install and initialize guest RAM. */
```

Comment:
See above my comment about choosing one term and sticking to it. I would prefer "initializing".

Reply:
done.